app/assets/stylesheets/partials/_govuk_radios.scss

```diff
@@ -0,0 +1,271 @@
+}
+
+.govuk-radios__divider {
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+  width: 40px;
+  margin-bottom: 10px;
+  text-align: center
+}
+
+@media print {
+  .govuk-radios__divider {
+    font-family: sans-serif
+  }
+}
+
+@media (min-width: 40.0625em) {
+  .govuk-radios__divider {
+    font-size: 19px;
+    font-size: 1.1875rem;
+    line-height: 1.31579
+  }
+}
+
+@media print {
+  .govuk-radios__divider {
+    font-size: 14pt;
+    line-height: 1.15
+  }
+}
+
+@media print {
+  .govuk-radios__divider {
+    color: #000
+  }
+}
+
+.govuk-radios__conditional {
+  margin-bottom: 15px;
+  margin-left: 18px;
+  padding-left: 33px;
+  border-left: 4px solid #b1b4b6
+}
+
+@media (min-width: 40.0625em) {
+  .govuk-radios__conditional {
+    margin-bottom: 20px
+  }
+}
+
+.js-enabled .govuk-radios__conditional--hidden {
+  display: none
+}
+
+.govuk-radios__conditional > :last-child {
+  margin-bottom: 0
+}
+
+.govuk-radios--small .govuk-radios__item {
+  min-height: 0;
+  margin-bottom: 0;
+  padding-left: 34px;
+  float: left
+}
+
+.govuk-radios--small .govuk-radios__item:after {
+  content: "";
+  display: block;
+  clear: both
+}
+
+.govuk-radios--small .govuk-radios__input {
+  left: -10px
+}
+
+.govuk-radios--small .govuk-radios__label {
+  margin-top: -2px;
+  padding: 13px 15px 13px 1px;
+  float: left
+}
+
+@media (min-width: 40.0625em) {
+  .govuk-radios--small .govuk-radios__label {
+    padding: 11px 15px 10px 1px
+  }
+}
+
+.govuk-radios--small .govuk-radios__label:before {
+  top: 8px;
+  width: 24px;
+  height: 24px
+}
+
+.govuk-radios--small .govuk-radios__label:after {
+  top: 15px;
+  left: 7px;
+  border-width: 5px
+}
+
+.govuk-radios--small .govuk-radios__hint {
+  padding: 0;
+  clear: both;
+  pointer-events: none
+}
+
+.govuk-radios--small .govuk-radios__conditional {
+  margin-left: 10px;
+  padding-left: 20px;
+  clear: both
+}
+
+.govuk-radios--small .govuk-radios__divider {
+  width: 24px;
+  margin-bottom: 5px
+}
+
+.govuk-radios--small .govuk-radios__item:hover .govuk-radios__input:not(:disabled) + .govuk-radios__label:before {
+  -webkit-box-shadow: 0 0 0 10px #b1b4b6;
+  box-shadow: 0 0 0 10px #b1b4b6
+}
+
+.govuk-radios--small .govuk-radios__item:hover .govuk-radios__input:focus + .govuk-radios__label:before {
+  -webkit-box-shadow: 0 0 0 4px #fd0, 0 0 0 10px #b1b4b6;
+  box-shadow: 0 0 0 4px #fd0, 0 0 0 10px #b1b4b6
+}
+
+@media (hover: none), (pointer: coarse) {
+  .govuk-radios--small .govuk-radios__item:hover .govuk-radios__input:not(:disabled) + .govuk-radios__label:before {
+    -webkit-box-shadow: initial;
+    box-shadow: initial
+  }
+  .govuk-radios--small .govuk-radios__item:hover .govuk-radios__input:focus + .govuk-radios__label:before {
+    -webkit-box-shadow: 0 0 0 4px #fd0;
+    box-shadow: 0 0 0 4px #fd0
+  }
+}
```

Comment:
Has this just been ripped from govuk frontend? I can't wait to get rid of it haha but it'll have to wait until we migrate. It's so weird seeing a mix of old and new classes though